sssd_test_framework/misc/__init__.py

```diff
@@ -176,6 +176,20 @@ def seconds_to_timespan(seconds: int) -> str:
 
     return f"{d:02d}:{h:02d}:{m:02d}:{s:02d}:00"
 
+def ip_to_zone_name(address: str) -> str:
+    """
+    Get PTR zone name from given address.
+
+    :param address: Address.
+    :type address: str
+    :return: PTR zone name.
+    :rtype: str
+    """
+    parts = address.split(".")
+    if parts < 4:
```

Comment:
I would like to se it the other way around

```
if parts == 4:
        return f"{'.'.join(parts[1:])}.in-addr.arpa"
```
To avoid issue with 1.2.3.4.5